src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java

```diff
@@ -958,6 +961,12 @@ private CSVWriter writeComments(HttpServletResponse servletResponse, List<String
     return properties;
   }
 
+  /** returns a new geometry out of the rounded coordinates of the given one */
+  private Geometry roundCoordsOfGeomToSevenDigits(Geometry oldGeom) {
+    PrecisionModel pm = new PrecisionModel(1E7);
+    return GeometryPrecisionReducer.reduce(oldGeom, pm);
```

Comment:
a few comments:

* creating a new `PrecisionModel` for every time this method is called could introduce unwanted overhead here
* why not use a `GeometryPrecisionReducer` directly (`var gpr = new GeometryPrecisionReducer(… … return gpr.reduce(oldGeom);`), instead of using the static convenience method with a `PrecisionModel`?
* AFAICS, the `GeometryPrecisionReducer` will perform additional checks like eliminating duplicate vertices and checking polygons for validity:
  
  >  By default the reduced result is topologically valid (i.e. Geometry.isValid() is true). To ensure this a polygonal geometry is reduced in a topologically valid fashion (technically, by using snap-rounding). Note that this may change polygonal geometry structure (e.g. two polygons separated by a distance below the specified precision will be merged into a single polygon). 
  
  This could be problematic in two ways: it might introduce a performance penalty and it might alter the output in potentially unwanted ways (in general we want to return the OSM features' geometries AS closely to as they are mapped in OSM). Why not activate the [_pointwise_](https://locationtech.github.io/jts/javadoc/org/locationtech/jts/precision/GeometryPrecisionReducer.html#setPointwise-boolean-) and [_non-collapsing_](https://locationtech.github.io/jts/javadoc/org/locationtech/jts/precision/GeometryPrecisionReducer.html#setRemoveCollapsedComponents-boolean-) mode of the `GeometryPrecisionReducer`?